core/clause_7_core.adoc

```diff
@@ -200,11 +200,11 @@ The following section describes a method to retrieve the available processes off
 
 ==== Operation
 
-include::requirements/core/REQ_process-collection-op.adoc[]
```

Comment:
I don't understand this change.

Reply:
Maybe it is a mistake but, I have noticed when trying to produce the output document using asciidoctor that the file was not named REQ_process-collection-op.adoc but REQ_job-collection-op.adoc, hence I have made the modification to include the correct filename.

Reply:
Right, the mistake is in clause_10_job_collection.adoc. The software I am using seems to be a bit more forgiving regarding wrong includes..